src/api/forms/toggle-check-box-mixin.js

```diff
@@ -0,0 +1,19 @@
+export default {
```

Comment:
would name this toggle-mixin or form-toggle-mixin as it is used by at least check box and switch.

Reply:
Let's file a followup for this and assign to @nikhilmandlik